polaris-service/src/main/java/org/apache/polaris/service/config/PolarisApplicationConfig.java

```diff
@@ -146,6 +147,15 @@ public void setFeatureConfiguration(Map<String, Object> featureConfiguration) {
     this.configurationStore = new DefaultConfigurationStore(featureConfiguration);
   }
 
+  @JsonProperty("maxDocumentBytes")
+  public void setMaxDocumentBytes(long maxDocumentBytes) {
+    this.maxDocumentBytes = maxDocumentBytes;
+  }
+
+  public long getMaxDocumentBytes() {
+    return maxDocumentBytes;
+  }
```

Comment:
Checking a bit more about class `StreamReadConstraints`.  `-1` is the right default value, which means no limit.
```
        public Builder maxDocumentLength(long maxDocLen) {
            if (maxDocLen <= 0L) {
                maxDocLen = -1L;
            }

            this.maxDocLen = maxDocLen;
            return this;
        }
```

Reply:
Thank you for pulling this up, this actually makes me more strongly interested in 0. Please see my new comment below.

Reply:
I also find 0-as-unlimited a little confusing. I agree that if we make the default -1 it might imply that 0 is different from -1, but I also think a default of 0 is going to raise some questions.

Maybe we should just disallow 0, since `StreamReadConstraints` doesn't support what 0 would intuitively mean ("reject all requests")

Reply:
From the code from StreamReadConstraints, that means both 0 and -1 are considered no limits. But I think it does make sense to have -1 for unlimited and disallow 0.

Reply:
Ok I updated it to default to -1, explained what -1 is, and restricted 0 with an exception/useful comment.